src/System Application/App/Page Style/PageStyle.al

```diff
@@ -0,0 +1,49 @@
+enum 50000 "Page Style"
+{
+    Extensible = false;
+
+    value(0; None)
+    {
+        Caption = 'None', Comment = 'None';
+    }
+    value(1; Standard)
+    {
+        Caption = 'Standard', Comment = 'Standard';
+    }
+    value(2; StandardAccend)
+    {
+        Caption = 'StandardAccent', Comment = 'Blue';
```

Comment:
Captions should be user friendly. So we should add some spaces. --> "Standard Accent"

Reply:
OK then we would need codeunits to translate the values.
If not implemented through a codeunit, I would need to lock the captions, as these vales should be constant.

The Intention would be an enum to be used for StyleExpr-Text variables in pages. Maybe you have a proper solution but I intended to use the enum directly.

I convert the PR to DRAFT because from this point there are too much questions from my point of view.
All imputs to improve are appreciated.

Reply:
Let's stick to the property values as defined here: https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-style-property?wt.mc_id=d365bc_inproduct_alextension

Reply:
on second thought, you're right. but in that case we should lock the captions, right?

Reply:
Typo corrected and changed captions to locked